assets/js/modules/adsense/dashboard/adsense-module-status.js

```diff
@@ -307,7 +307,7 @@ class AdSenseModuleStatus extends Component {
 						</div>
 					) }
 
-					{ showInProcess && (
+					{ showInProcess && 'account-connected' !== accountStatus && (
```

Comment:
@adamsilverstein Can you clarify why you added this condition here?

Reply:
when the status is 'account-connected' we want to show the `AdSenseSetupInstructions` component instead of the `AdSenseInProcessStatus` - thats what this conditional is doing. alternately, it could be added above to `showInProcess` itself.

Reply:
That makes sense, however would we ever want to show both `AdSenseInProcessStatus` and `AdSenseSetupInstructions`? In the original code (before the refactoring), we were only showing one of them, and `AdSenseInProcessStatus` was relying on three specific statuses: https://github.com/google/site-kit-wp/blob/39e8d2de110dd34c8c6711239b3018b41d1545b8/assets/js/modules/adsense/dashboard/adsense-module-status.js#L109

cc @tofumatt

Reply:
I realised we don't. The issue seems to be the original `AdSenseSetupInstructions` has code that covers `accountStatus` not being `'account-connected'`, but that shouldn't happen. So this is fine.